apps/backend/src/users/types.ts

```diff
@@ -0,0 +1,31 @@
+// TODO: Probably want these types to be available to both the frontend and backend in a "common" folder
+export enum Status {
+  MEMBER = 'Member',
+  RECRUITER = 'Recruiter',
+  ADMIN = 'Admin',
+  ALUMNI = 'Alumni',
+  APPLICANT = 'Applicant',
+}
+
+export enum Team {
+  SFTT = 'Speak For The Trees',
+  CONSTELLATION = 'Constellation',
+  JPAL = 'J-PAL',
+  BREAKTIME = 'Breaktime',
+  GI = 'Green Infrastructure',
+  CI = 'Core Infrastructure',
+}
+
+export enum Role {
+  DIRECTOR_OF_ENGINEERING = 'Director of Engineering',
+  DIRECTOR_OF_PRODUCT = 'Director of Product',
+  DIRECTOR_OF_FINANCE = 'Director of Finance',
+  DIRECTOR_OF_MARKETING = 'Director of Marketing',
+  DIRECTOR_OF_RECRUITMENT = 'Director of Recruitment',
+  CO_DIRECTOR_OF_OPERATIONS = 'Co-Director of Operations',
```

Comment:
:wrench: can we take out the `CO_` prefix from ops and events? It's only fairly recent that we started having co-directors (so that we'll probably have just one co-director of ops in a year or two again) and we could theoretically have multiple people with the same position for most of the other director-level positions as well. And some teams have co-tech leads too so I think it'll be better to just ignore the "co-" part and we'll know if there are co-directors if there are multiple people with the same role